src/cookieSync.js

```diff
@@ -18,9 +20,33 @@ function triggerPixel(url) {
   img.src = url;
 }
 
+function insertUserSyncIframe(url) {
+  let iframe = document.createElement('iframe');
+  iframe.src = url;
+  insertElement(iframe);
```

Comment:
pass other parameters to insertElement , insertElement(elm, doc, 'body') is recommended

Reply:
Done